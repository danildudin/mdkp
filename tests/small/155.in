15 4
32 50 8 6 10 43 5 77 32 34 90 77 38 36 67 
33 66 4 99 73 25 80 70 45 94 22 67 50 71 24 213
84 98 93 59 45 30 17 12 43 43 61 9 46 71 81 296
83 10 67 51 59 34 73 50 49 14 80 17 25 90 15 435
36 75 87 45 25 15 62 76 35 47 89 19 68 100 34 404
