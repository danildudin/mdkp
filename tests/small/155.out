362
6
2 5 7 10 11 14 
